src/Library/WizardControl.cs

```diff
@@ -43,7 +43,10 @@ public partial class WizardControl :
 		private WizardClassicStyle classicStyle = WizardClassicStyle.AeroStyle;
 		private Point formMoveLastMousePos;
 		private bool formMoveTracking;
-		private Form parentForm;
+		private ContainerControl parentControl;
+		private Form parentAsForm => parentControl as Form;
```

Comment:
Please remove these two private properties that really just hide the function of checking the control type. See other comments for handling.

Reply:
Fixed, I accidentally created a new pull request. This one can be rejected.